cardano-cli/src/Cardano/CLI/Types/Common.hs

```diff
@@ -127,6 +129,14 @@ data ProposalHashSource
   | ProposalHashSourceHash (L.SafeHash Crypto.StandardCrypto L.AnchorData)
   deriving Show
 
+newtype AnchorUrl = AnchorUrl
```

Comment:
This is fine here for now but we should move them to `cardano-api` in a follow up PR.

Reply:
Sounds good :+1: Created follow-up issue to keep track: https://github.com/input-output-hk/cardano-api/issues/301